app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,38 +20,3 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
-
-.gem-c-breadcrumbs--collapse-on-mobile {
-  @include govuk-media-query($until: tablet) {
-    .govuk-breadcrumbs__list {
-      display: flex;
-    }
-
-    .govuk-breadcrumbs__list-item:not(:last-child):not(:first-child) {
-      display: none;
-    }
-
-    .govuk-breadcrumbs__list-item {
```

Comment:
The govuk-frontend change doesn't include the touch target change as there were some questions about that. It might be worth leaving that in if we can, until we get an approach sorted out with the design system?

Reply:
Good catch. I kept the definitions that cover touch target change and tested the examples again.